nwjs: 0.100.1 -> 0.102.1

r-ryantm opened the pull request:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from https://repology.org/project/nwjs/versions.


meta.description for nwjs is: App runtime based on Chromium and node.js

meta.homepage for nwjs is: https://nwjs.io/




###### Updates performed
- Version update

###### To inspect upstream changes





###### Impact

<b>Checks done</b>

---

- built on NixOS
- The tests defined in `passthru.tests`, if any, passed
- found 0.101.2 with grep in /nix/store/05wvi8raswbwsabfqf8f7jb5ffv91zn9-nwjs-0.101.2
- found 0.101.2 in filename of file in /nix/store/05wvi8raswbwsabfqf8f7jb5ffv91zn9-nwjs-0.101.2

---

<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
12 total rebuild path(s)

12 package rebuild(s)

First fifty rebuilds by attrpath

betaflight-configurator
emuflight-configurator
gridtracker
icestudio
inav-configurator
nwjs
nwjs-sdk
onlykey
pinegrow
pinegrow6
popcorntime
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from the cache**:
```
nix-store -r /nix/store/05wvi8raswbwsabfqf8f7jb5ffv91zn9-nwjs-0.101.2 \
  --option binary-caches 'https://cache.nixos.org/ https://nixpkgs-update-cache.nix-community.org/' \
  --option trusted-public-keys '
  nixpkgs-update-cache.nix-community.org-1:U8d6wiQecHUPJFSqHN9GSSmNkmdiFW7GW7WNAnHW0SM=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The nixpkgs-update cache is only trusted for this store-path realization.)
For the cached download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A nwjs https://github.com/r-ryantm/nixpkgs/archive/26d21b8ac3a35cc150073f8dfe6f4283a4cfbb2d.tar.gz
```
Or:
```
nix build github:r-ryantm/nixpkgs/26d21b8ac3a35cc150073f8dfe6f4283a4cfbb2d#nwjs
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/05wvi8raswbwsabfqf8f7jb5ffv91zn9-nwjs-0.101.2
ls -la /nix/store/05wvi8raswbwsabfqf8f7jb5ffv91zn9-nwjs-0.101.2/bin
```

---

</details>
<br/>



### Pre-merge build results

We have automatically built all packages that will get rebuilt due to
this change.

This gives evidence on whether the upgrade will break dependent packages.
Note sometimes packages show up as _failed to build_ independent of the
change, simply because they are already broken on the target branch.

## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review`

---
### `x86_64-linux`
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>nwjs-sdk</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 10 packages built:</summary>
  <ul>
    <li>betaflight-configurator</li>
    <li>emuflight-configurator</li>
    <li>gridtracker</li>
    <li>icestudio</li>
    <li>inav-configurator</li>
    <li>nwjs</li>
    <li>onlykey</li>
    <li>pinegrow</li>
    <li>pinegrow6</li>
    <li>popcorntime</li>
  </ul>
</details>

---

###### Maintainer pings

cc @MikaelFangel for [testing](https://github.com/ryantm/nixpkgs-update/blob/main/doc/nixpkgs-maintainer-faq.md#r-ryantm-opened-a-pr-for-my-package-what-do-i-do).

> [!TIP]
> As a maintainer, if your package is located under `pkgs/by-name/*`, you can comment **`@NixOS/nixpkgs-merge-bot merge`** to automatically merge this update using the [`nixpkgs-merge-bot`](https://github.com/NixOS/nixpkgs-merge-bot).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

r-ryantm commented:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from https://repology.org/project/nwjs/versions.


meta.description for nwjs is: App runtime based on Chromium and node.js

meta.homepage for nwjs is: https://nwjs.io/




###### Updates performed
- Version update

###### To inspect upstream changes





###### Impact

<b>Checks done</b>

---

- built on NixOS
- The tests defined in `passthru.tests`, if any, passed
- found 0.102.0 with grep in /nix/store/dh8fw81rndv149jcsgc11yfj8ivx5cz0-nwjs-0.102.0
- found 0.102.0 in filename of file in /nix/store/dh8fw81rndv149jcsgc11yfj8ivx5cz0-nwjs-0.102.0

---

<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
12 total rebuild path(s)

12 package rebuild(s)

First fifty rebuilds by attrpath

betaflight-configurator
emuflight-configurator
gridtracker
icestudio
inav-configurator
nwjs
nwjs-sdk
onlykey
pinegrow
pinegrow6
popcorntime
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from the cache**:
```
nix-store -r /nix/store/dh8fw81rndv149jcsgc11yfj8ivx5cz0-nwjs-0.102.0 \
  --option binary-caches 'https://cache.nixos.org/ https://nixpkgs-update-cache.nix-community.org/' \
  --option trusted-public-keys '
  nixpkgs-update-cache.nix-community.org-1:U8d6wiQecHUPJFSqHN9GSSmNkmdiFW7GW7WNAnHW0SM=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The nixpkgs-update cache is only trusted for this store-path realization.)
For the cached download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A nwjs https://github.com/r-ryantm/nixpkgs/archive/c995493df9a1f78a8b2ba9a82502d5ad983dac6b.tar.gz
```
Or:
```
nix build github:r-ryantm/nixpkgs/c995493df9a1f78a8b2ba9a82502d5ad983dac6b#nwjs
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/dh8fw81rndv149jcsgc11yfj8ivx5cz0-nwjs-0.102.0
ls -la /nix/store/dh8fw81rndv149jcsgc11yfj8ivx5cz0-nwjs-0.102.0/bin
```

---

</details>
<br/>



### Pre-merge build results

We have automatically built all packages that will get rebuilt due to
this change.

This gives evidence on whether the upgrade will break dependent packages.
Note sometimes packages show up as _failed to build_ independent of the
change, simply because they are already broken on the target branch.

## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review`

---
### `x86_64-linux`
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>nwjs-sdk</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 10 packages built:</summary>
  <ul>
    <li>betaflight-configurator</li>
    <li>emuflight-configurator</li>
    <li>gridtracker</li>
    <li>icestudio</li>
    <li>inav-configurator</li>
    <li>nwjs</li>
    <li>onlykey</li>
    <li>pinegrow</li>
    <li>pinegrow6</li>
    <li>popcorntime</li>
  </ul>
</details>

---

###### Maintainer pings

cc @MikaelFangel for [testing](https://github.com/ryantm/nixpkgs-update/blob/main/doc/nixpkgs-maintainer-faq.md#r-ryantm-opened-a-pr-for-my-package-what-do-i-do).

> [!TIP]
> As a maintainer, if your package is located under `pkgs/by-name/*`, you can comment **`@NixOS/nixpkgs-merge-bot merge`** to automatically merge this update using the [`nixpkgs-merge-bot`](https://github.com/NixOS/nixpkgs-merge-bot).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

r-ryantm committed:
nwjs: 0.100.1 -> 0.102.1

MikaelFangel commented on pkgs/development/tools/nwjs/default.nix:
```diff
@@ -108,7 +108,7 @@ stdenv.mkDerivation {
           "sdk-x64" = "sha256-nD1MYEm+4sq4MHkiOI46IgM6oDWpcyVWl3Yis0beEIg=";
           "ia32" = "sha256-TfAw7+z/I3eYRn20DN0sR9moqD23UG7nvQbeXQDNDcg=";
-          "x64" = "sha256-EqLESL4zVdBzFvhPgXfcIwSVNNBVIMy1SdCu6eNU96Q=";
+          "x64" = "sha256-MuHWUqGc9WM0clgcdX/8BKJyaNVjMS9PUdPsGiQTpHg=";
```
I think it would be fine if we disabled the updates because I'm currently not in a position to write an update script for nwjs.

wolfgangwalther committed:
nwjs: disable auto-updates

As long as we don't have an update script that handles the 4 hashes
correctly, this won't help.
